lumicks/pylake/population/tests/test_hmm_algos.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 import pytest
 
-from lumicks.pylake import HiddenMarkovModel, GaussianMixtureModel
```

Comment:
Hmm. I'm surprised this dependency that is not needed wasn't picked up by flake earlier.

Reply:
maybe we don't enforce `flake8` on the tests? it was only recently we enforced `black` there

Reply:
We should be enforcing it though. If I look at the config, we only exclude that warning for `__init.py__`. I'll look into it. Not part of this PR anyhow.
```
    # Ignore imported but not used for __init__.py files.
    __init__.py:F401
```